apps/test/unit/lib/script-editor/StageDescriptionsTest.js

```diff
@@ -56,20 +53,25 @@ describe('StageDescriptions', () => {
     wrapper.find('button').simulate('click');
     assert.equal(wrapper.state('collapsed'), false);
 
-    // button replaced by a div
-    assert.equal(wrapper.childAt(1).children().length, 1);
-    assert.equal(
-      wrapper
-        .childAt(1)
-        .childAt(0)
-        .type(),
-      'div'
-    );
+    const button = wrapper.childAt(1).childAt(0);
+    assert.equal(button.type(), 'button');
+    assert.include(button.text(), 'Collapse');
+
+    // button followed by a div
+    assert.equal(wrapper.childAt(1).children().length, 2);
+    const descriptions = wrapper.childAt(1).childAt(1);
+    assert.equal(descriptions.type(), 'div');
 
     assert.equal(
-      wrapper.find('button').text(),
+      descriptions.find('button').text(),
       'Import from Curriculum Builder'
     );
+
+    // collapses after subsequent button click
+    button.simulate('click');
+    assert.equal(wrapper.state('collapsed'), true);
+    assert.include(button.text(), 'Collapse');
```

Comment:
good catch! it looks like this was only passing because the test had a stale reference to the button. I've updated the text and fixed the test to make it pass.